e2e-tests/content-strings/fields/insurance/policy/index.js

```diff
@@ -259,14 +259,36 @@ export const POLICY_FIELDS = {
         FORM_TITLE: POLICY_FORM_TITLES.BROKER,
       },
     },
-    [FULL_ADDRESS]: {
-      LABEL: "Broker's address",
+    [IS_BASED_IN_UK]: {
+      LABEL: 'Is the broker based in the UK?',
       SUMMARY: {
-        TITLE: "Broker's address",
+        TITLE: 'Is the broker based in the UK?',
         FORM_TITLE: POLICY_FORM_TITLES.BROKER,
       },
-      MAXIMUM: MAXIMUM_CHARACTERS.FULL_ADDRESS,
     },
+    [POSTCODE]: {
+      LABEL: 'Postcode',
+      SUMMARY: {
+        TITLE: 'Postcode',
+        FORM_TITLE: POLICY_FORM_TITLES.BROKER,
+      },
+    },
+    [BUILDING_NUMBER_OR_NAME]: {
+      LABEL: 'Building number or name',
+      SUMMARY: {
+        TITLE: 'Building number or name',
+        FORM_TITLE: POLICY_FORM_TITLES.BROKER,
+      },
+    },
+    // TODO: EMS-3975 - move FULL_ADDRESS to new form
```

Comment:
Can the code be in the Jira ticket please and just leave the Todo with the ticket number for the all instances.